src/SonarQube.Client/Services/ISonarQubeClient.cs

```diff
@@ -0,0 +1,90 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Threading;
+using System.Threading.Tasks;
+using SonarQube.Client.Models;
+
+namespace SonarQube.Client.Services
+{
+    public interface ISonarQubeClient
+    {
+        /// <summary>
```

Comment:
Empty comment.

We don't normally use comments, so I'd remove them from here as well. They don't bring any value here, apart from remark on `GetRoslynExportProfileAsync`.